meta/main.yml

```diff
@@ -6,7 +6,6 @@ galaxy_info:
   company: Red Hat, Inc.
   license: BSD-3-Clause
   min_ansible_version: "2.9"
-  github_branch: main
```

Comment:
This is also not supported in ansible-core-2.14, ansible-lint 6.x ?

Reply:
> This is also not supported in ansible-core-2.14, ansible-lint 6.x ?

Right.  It violates the schema of meta/main.yml - now enforced by ansible-lint 6.x